### DISCUSSION TITLE
Multiple output with single input

### DISCUSSION THREAD

**Discussion by techmech46** [cyberbotics/webots#4114] [Q&A]
I am using WEBOT simulator. I am trying to run NAO robot. I am using C language. In nao.demo.c, we can move robot forward by clicking forward button, we can move robot left by clicking left button and so on.
Can't we get 2 motion in a single click. Like I want to move robot forward then it will move left. But I want both motion to be done by only one click.
Can we do this?

**Comment by omichel** [comment]
Yes, you can do this. You should edit the [nao_demo.c](https://github.com/cyberbotics/webots/blob/master/projects/robots/softbank/nao/controllers/nao_demo/nao_demo.c) controller and re-program it to execute sequentially several motions instead of one.

**Reply by techmech46** [reply]
Can you give me an example for the same.

Like I have 

Case wb_keyword_left;
          Start_motion(side_step_left);
Break;
Case wb_keyword_right
         Start_motion(side_step_right);
Break;

What can I do to get both motion in a single click?


I have done this, but still I am getting only one motion.

Case wb_keyword_left;
          Start_motion(side_step_left);

         Start_motion(side_step_right);
Break;
